pkg/daemon/daemon.go

```diff
@@ -323,8 +324,9 @@ func (d *Daemon) sharedErofsUmount(rafs *Rafs) error {
 		return errors.Wrapf(err, "unbind blob %s", d.ID())
 	}
 	domainID := rafs.Annotations[AnnoFsCacheDomainID]
+	fscaheID := rafs.Annotations[AnnoFsCacheID]
```

Comment:
It's better to handle the absence of fscacheid. Moreover, should the name be consistent? I saw it is named blobID in function `UnbindBlob` body

Reply:
fscacheID will be set with domainID together, it can not be absent. fscacheID will used for generate fscache cookie key for bootstrap, and we need it when delete fscache cache file for bootstrap. Already add comment for this.